aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -118,6 +118,67 @@ TORCH_META_FUNC(argmin)
 
 } // namespace meta
 
+namespace meta {
+
+TORCH_META_FUNC(aminmax)
+(const Tensor& self, c10::optional<int64_t> dim_opt, bool keepdim) {
+  DimVector shape;
+  if (dim_opt.has_value()) {
+    auto dim = maybe_wrap_dim(dim_opt.value(), self.ndimension());
+    native::zero_numel_check_dims(self, dim, "aminmax");
+    shape = get_reduction_shape(self, dim, keepdim);
+  } else {
+    TORCH_CHECK(
+        self.numel() > 0,
+        "minmax(): cannot compute minmax over an empty dimension as the "
+        "operation has no identity.");
+    if (keepdim) {
+      shape = DimVector(self.ndimension(), 1);
+    }
+  }
+  const auto options = self.options();
+  const auto out0 = this->maybe_get_output(0);
+  const auto out1 = this->maybe_get_output(1);
+  this->set_output(
+      0,
+      shape,
+      options.dtype(out0.defined() ? out0.scalar_type() : self.scalar_type()));
```

Comment:
we don't support `out` of the different types (in fact, in the implementation there are checks to make sure that `min` and `max` are the same type. It means that you don't have to do `maybe_get_output`/`set_output`, just `set_output` is enough. 
cc @ezyang, we typically shouldn't need maybe_get_output in the META_FUNC, right?

Reply:
Wouldn't I still need to get one output to compute the dtype for set_output? Otherwise I will always set it to the input's dtype.

Reply:
Yes, you only need it if you have behavior where you accommodate non-standard out dtype and handle the casting for them directly. If you require the out dtype to exactly match what the functional dtype was, only set_output is needed.